src/developer/web-api/tracker.md

```diff
@@ -3249,7 +3236,7 @@ Table: Payload
 |description|A description of the working list.|"description":"for listing all events assigned to me".|
 |program|The uid of the program.|"program" : "a3kGcGDCuk6"|
 |programStage|The uid of the program stage.|"programStage" : "a3kGcGDCuk6"|
-|eventQueryCriteria|Object containing parameters for querying, sorting and filtering events.|"eventQueryCriteria": {     "organisationUnit":"a3kGcGDCuk6",     "status": "COMPLETED",     "createdDate": {       "from": "2014-05-01",       "to": "2019-03-20"     },     "dataElements": ["a3kGcGDCuk6:EQ:1", "a3kGcGDCuk6"],     "filters": ["a3kGcGDCuk6:EQ:1"],     "programStatus": "ACTIVE",     "ouMode": "SELECTED",     "assignedUserMode": "PROVIDED",     "assignedUsers" : ["a3kGcGDCuk7", "a3kGcGDCuk8"],     "followUp": false,     "trackedEntityInstance": "a3kGcGDCuk6",     "events": ["a3kGcGDCuk7", "a3kGcGDCuk8"],     "fields": "eventDate,dueDate",     "order": "dueDate:asc,createdDate:desc"   }|
```

Comment:
Looks like we removed a property (and not replace it with `trackedEntity` ) . Technically a breaking change right 🤔

Reply:
I don't know if this was ever in there 🤷🏻 https://github.com/dhis2/dhis2-core/commits/master/dhis-2/dhis-api/src/main/java/org/hisp/dhis/programstagefilter/EventQueryCriteria.java has not changed in a long time. Maybe this was just a mistake in the docs